libs/README.rst

```diff
@@ -76,13 +76,28 @@ notwendig.)
 macOS
 ^^^^^
 Unter MacOS ist die Einrichtung relativ einfach und bedarf nur der oben genannten Voreinstellungen.
-Es wird der von Apple ausgelieferte clang compiler verwendet.
+Es wird der von Apple ausgelieferte clang compiler verwendet. 
+ Beispiel: Innerhalb von macOS /Users/governikus/AusweisApp2 befindet sich der Quellcode.
+::
 
+   $ cd $HOME/governikus
+   $ mkdir build
+   $ cd build
+   $ cmake -DCMAKE_BUILD_TYPE=release ../AusweisApp2/libs
+   $ make
+
+=======
 Beispiel: Innerhalb von /Users/governikus/AusweisApp befindet sich der Quellcode.
 
+
+Linux / Unix
+^^^^^^^^^^^^
+Die Einrichtung unter Linux/Unix ist einfach und erfordert nur die oben genannten Einstellungen. Die meisten Distributionen haben den C Compiler bereits vorinstalliert, aber er kann auch nachträglich installiert werden.
+ Beispiel: Innerhalb von Linux/Unix /home/governikus/AusweisApp2 befindet sich der Quellcode 
+ 
 ::
 
-   $ cd /Users/governikus
+   $ cd $HOME/governikus
```

Comment:
$HOME is usually /home/$USER and governikus is the $USER part. So it's redudant and would result in a tree of

/home/governikus/governikus  
/home/governikus/governikus/build/

Assuming the path /home/governikus/AusweisApp2 is given. The cmake won´t find the directory.

I don't own a Mac but I assume it to be similar.

Reply:
i will fix that